FlashEnglish/Sources/Views/Home/HomeView.swift

```diff
@@ -8,22 +8,34 @@
 import SwiftUI
 
 struct HomeView: View {
+
+
+
+
+
+
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Masaya1582/FlashEnglish/blob/0e74ba265c47132c04f133da09675f0b9e81baab/FlashEnglish/Sources/Views/Home/HomeView.swift#L16"></span>Limit vertical whitespace to a single empty line; currently 6<br />
<code>vertical_whitespace</code> <code>HomeView.swift:16</code></td>
    </tr>
  </tbody>
</table>